src/components/Button/Button.vue

```diff
@@ -361,7 +361,7 @@ export default {
 
 	methods: {
 		getText() {
-			return this.$slots.default ? this.$slots.default[0].text.trim() : null
+			return this.$slots.default && this.$slots.default[0] && this.$slots.default[0].text ? this.$slots.default[0].text.trim() : null
```

Comment:
Probably better this.$slots?.default[0]?.text?

Reply:
Fixed

Reply:
Regression https://github.com/nextcloud/nextcloud-vue/pull/2547

Reply:
@nickvergessen @JuliaKirschenheuter 
I think the correct expression would be:

`this.$slots?.default?.[0]?.text?.trim()`

Reply:
That sadly didn't work 🤷‍♂️

Reply:
Hmm. Okay, but did you noticed the `?.` directly behind of `default` ? That wasn't included in the commit. Though it is needed when `default` can be null/undefined.